book/appendix/wallaroo-command-line-options.md

```diff
@@ -4,66 +4,100 @@ Every Wallaroo option exposes a set of command-line options that are used to con
+`--application-module` specifies the _name_ that machida will attempt to import as the Python Wallaroo application file. For example, if you write a Python Wallaroo application and save it as `my_application.py`, then you should provide that name to machida as `--application-module my_application`.
+The application should be available to import from the location where `machida` is run. You can achieve this by:
+1. Having `machida` and `my_application.py` in the same directory
+2. Adding path to `my_application.py` to your ENV's `PYTHONPATH`
+3. Adding the current directory path (`.`) to your ENV's `PYTHONPATH` and executing the `machida` command from the directory where `my_application.py` is located.
 
 ### Multi-Worker Setup
 
 A Wallaroo application can be distributed over multiple Wallaroo processes, or "workers". When a Wallaroo application first spins up, one of these workers plays the role of the "initializer", a temporary role that only has significance during initialization (after that, the cluster is decentralized).
 
 We specify that a worker will play the role of initializer via the `--cluster-initializer` flag. We specify the total number of workers at startup (including the initializer) via the `--worker-count` parameter.
 
-Workers communicate with each other over two channels, a control channel and a data channel. We need to tell every worker which addresses the initializer is listening on for each of these channels. `--control` specifies the initializer's control address and `--data` specifies the initializer's data address.
+Workers communicate with each other over two channels: a control channel and a data channel. We need to tell every worker which addresses the initializer is listening on for each of these channels. `--control` specifies the initializer's control address and `--data` specifies the initializer's data address.
 
 Remember that multi-worker runs require that you distribute the same binary to all machines that are involved. Attempting to connect different binaries into a multi-worker cluster will fail.
 
 #### Example Single-Worker Run
 
-In order to run a single worker for a Wallaroo app, we need to specify the input address (`-i`), the target output address (`-o`), and the metrics target address (`-m`). For example, using the Celsius Converter application (after navigating to `examples/pony/celsius` and compiling) we would run:
+In order to run a single worker for a Wallaroo app, we need to specify the input address (`--in`), the target output address for the TCP sink (`--out`), and the metrics target address (`--metrics`). For example, using the Celsius Converter application (after navigating to `examples/pony/celsius` and compiling) we would run:
 
 ```bash
-./celsius -i 127.0.0.1:6000 -o 127.0.0.1:5555 -m 127.0.0.1:5001
+./celsius --in 127.0.0.1:6000 --out 127.0.0.1:5555 --metrics 127.0.0.1:5001
 ```
 
 #### Example Multi-Worker Run
```

Comment:
This is covered in clustering. I don't think it makes sense to be here anymore.

Reply:
Should this be replaced with a link to the clustering docs?

Reply:
@SeanTAllen I assume by "this" you mean "Example Multi-Worker Run"? If this is a problem then we should open a new issue. It isn't part of the scope of this issue or PR.

Reply:
Yes. I'm ok with another issue.

Reply:
Shit, I accidentally deleted the comment @SeanTAllen is referring to. I asked for a separate issue covering removing this section.

My bad, sorry for the confusion.

Reply:
BASTARDO!